polling_stations/apps/data_finder/helpers/baked_data_helper.py

```diff
@@ -67,56 +50,45 @@ def __init__(self, elections_parquet_path: Path = None, **kwargs):
             )
         super().__init__(**kwargs)
 
-    def get_response_for_postcode(self, postcode: Postcode):
-        is_split, data_for_postcode = self.get_ballot_list(postcode)
-        data = {
-            "address_picker": is_split,
-            "addresses": [],
-        }
+    def get_response(self, postcode: Postcode, uprn: Optional[str] = None):
+        result = self.get_ballot_list(postcode, uprn)
 
-        if is_split:
-            data["addresses"] = data_for_postcode
-        else:
-            data["dates"] = self.ballot_list_to_dates(data_for_postcode)
+        if not result["address_picker"]:
+            result["ballots"] = self.get_full_ballots(result["ballot_ids"])
+            # TODO: how to set request_success if some requests failed here?
 
-        return data
+        return result
 
-    def get_full_ballots(self, ballot_data):
+    def get_full_ballots(self, ballot_ids):
         result = []
-        for ballot in ballot_data:
-            ballot_id = ballot["ballot_paper_id"]
+        for ballot_id in ballot_ids:
             url = ballot_paper_id_to_ee_url(ballot_id)
             req = session.get(url)
             result.append(req.json())
         return result
 
-    def ballot_list_to_dates(self, ballot_list):
```

Comment:
Removing this keeps the data format the same whether it came from `EEFetcher` or `LocalParquetElectionsHelper`. We are always just dealing with a list of ballot objects